spec/unit/controllers/internal/syslog_drain_urls_controller_spec.rb

```diff
@@ -12,280 +12,6 @@ module VCAP::CloudController
     let!(:binding_with_drain1) { ServiceBinding.make(syslog_drain_url: 'fish,finger', app: app_obj, service_instance: instance1) }
     let!(:binding_with_drain2) { ServiceBinding.make(syslog_drain_url: 'foobar', app: app_obj, service_instance: instance2) }
 
-    describe 'GET /internal/v4/syslog_drain_urls' do
```

Comment:
It seems like this PR removed the test coverage for the shared hostname_from_app_name method. It would be good to add the test cases to the v5 specs.